src/routes/index.tsx

```diff
@@ -67,13 +71,14 @@ export default component$(() => {
             </div>
             <div>
               <label for="bill-name" class="block mb-2 text-xl font-medium">
-                Number of members
+                Number of members (default is 3)
               </label>
               <input
                 type="number"
                 id="bill-name"
                 class="border text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5"
                 placeholder="Enter number of members"
+                min={2}
                 onInput$={(_, el) =>
                   (billData.memberCount = parseInt(el.value))
                 }
```

Comment:
Rather than writing "default is 3" in the label, what do you think about making the input controlled and show the value already there?

For example by adding `value={billData.memberCount}`

Reply:
Initially I had entered the value that was displayed, but it did not convince me 100%, but if you think it's better, I can correct.